server/src/pages/shot/controller.js

```diff
@@ -97,7 +97,7 @@ exports.changeShotExpiration = function (shot, expiration) {
       location.reload();
     };
   }
-  req.send(`id=${encodeURIComponent(shot.id)}&expiration=${encodeURIComponent(expiration)}`);
+  req.send(`id=${encodeURIComponent(shot.id)}&expiration=${encodeURIComponent(expiration)}&_csrf=${model.csrfToken}`);
```

Comment:
I like to use `encodeURIComponent()` everywhere even when I'm pretty sure it won't be needed.